### PR TITLE
Separate swipe per panel

### PR DESCRIPTION
**Pull request by catalin-iosif-code**
Swipe now operates on each individual panel. Also refactored gesture scripts into a module.

### CODE REVIEW

File: _dprhtml/js/chrome.js [hunk 1]
```diff
@@ -462,6 +462,10 @@ var DPR_Chrome = (function () {
     </div>`;
 
     $('#main-pane-container').append(`${html}`);
+      // Add event listeners for swipe gestures
+      document.getElementById('main-pane-container-section-' + sPos).addEventListener('touchstart', DPR_Gesture.touchStart, true);
```
Comment (parthopdas) [inline _dprhtml/js/chrome.js:466]: There is an API above to get the section ID - this is so we don't hardcode the prefix.

Also code appears out of alignment.

File: _dprhtml/js/swipe_gestures.js [hunk 2]
```diff
@@ -1,59 +1,82 @@
...
+    function runMatchingCommand(e) {
+      const cmd = Object.entries(__dprViewModel.commands).find(([_, x]) => x().matchGesture(e));
+      if (cmd && !cmd[1]().notImplemented && cmd[1]().canExecute && cmd[1]().visible) {
+        cmd[1]().execute(e);
+        event.preventDefault();
+        return;
+      }
+    }
+
+    return {
```
Comment (parthopdas) [inline _dprhtml/js/swipe_gestures.js:77]: Adding a module is a good idea. ❤️

File: _dprhtml/js/chrome.js (hunk 1 above)
Comment (catalin-iosif-code) [reply _dprhtml/js/chrome.js]: Changed to use API.

File: _dprhtml/js/swipe_gestures.js (hunk 2 above)
Comment (catalin-iosif-code) [reply _dprhtml/js/swipe_gestures.js]: Aligned code here as well.

**Review by parthopdas — APPROVED** [review]

**Pull request merged**